app/components/menu-card.hbs

```diff
@@ -0,0 +1,57 @@
+<div class="au-o-grid__item au-u-1-3@medium au-u-1-2@small">
```

Comment:
Nitpick: I would pass this in from the parent component since this seems highly specific to how it's used. If the 2 are always supposed to be used together (in which case the classes make sense) it's probably clearer to move it into a subfolder invoke it as `<Menu::Card>` or something.

Reply:
Okay so I ended up implementing both solutions at once here. Doesn't really matter, but it looks a bit cleaner now.